code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -175,38 +175,85 @@
 /mob/living/carbon/human/proc/commune()
 	set category = "Abilities"
 	set name = "Commune with creature"
-	set desc = "Send a telepathic message to an unlucky recipient."
+	set desc = "Send a telepathic message to a recipient."
+
+	var/obj/item/organ/external/rhand = src.get_organ("r_hand")
+	var/obj/item/organ/external/lhand = src.get_organ("l_hand")
+	if((!rhand || !rhand.is_usable()) && (!lhand || !lhand.is_usable()))
+		to_chat(src,"<span class='warning'>You can't communicate without the ability to use your hands!</span>")
+		return FALSE
+	if((lhand.is_stump()) && (rhand.is_stump()))
```

Comment:
Isn't this handled by `is_usable()`?

Reply:
Surprisingly, no. is_usable does not check for stumps. Which is why legcuffs check for stumps instead of usable.

Reply:
What. Oh my God.

Reply:
Hm, it might be a good idea to throw the is_stump() check into is_usable then.

Reply:
:agony:

Reply:
disregard it